### DISCUSSION TITLE
Surrendering without a castle is disabled in fheroes2 but not OG

### DISCUSSION THREAD

**Discussion by zenseii** [ihhub/fheroes2#8302] [General]
@oleg-derevenetz.

I just noticed that in fheroes2 surrendering is disabled when you don't own any castle, whereas in the OG you can surrender.

I believe you brought up the tactic of surrendering a hero in case you where able to re-claim a castle and then re-buy them as one of the reasons to surrender in a last stand fight, so maybe we should change back to the way OG works?

**Comment by oleg-derevenetz** [comment]
Well, technically, not having a castle is not a reason to disable the surrender, so I don't have an idea why it has been disabled, because even if you don't have a castle right now, you can capture it by another hero, etc.

**Reply by zenseii** [reply]
I agree. It smells like something pre-fheroes2 to be honest.

Considering that surrender hasn't really been a topic of discussion in fheroes2 until your recent AI improvements, I find it hard to believe this was done intentionally by us.

**Reply by oleg-derevenetz** [reply]
This is planned to be fixed in #8297.
